manifests/backend.pp

```diff
@@ -28,8 +28,16 @@
 #   Sensu backend configuration hash used to define backend.yml.
 # @param ssl_cert_source
 #   The SSL certificate source
```

Comment:
Under both `_content` and `_source` params put the note 'This parameter is mutually exclusive with X'

Reply:
Changes made